src/Control/RequestHandler.php

```diff
@@ -576,6 +576,17 @@ public function Link($action = null)
         return null;
     }
 
+    /**
+     * Get the absolute URL for this controller, including protocol and host.
+     *
+     * @param string $action See {@link Link()}
+     * @return string
+     */
+    public function AbsoluteLink($action = '')
```

Comment:
Can we guarantee the return type here?
```suggestion
    public function AbsoluteLink($action = ''): string
```

Reply:
<slight sarcasm> We'll never get there to full types with this approach </> :-D but I get the reason here.